.ci/es-snapshots/Jenkinsfile_build_es

```diff
@@ -157,6 +157,9 @@ def buildArchives(destination) {
       ./gradlew -p distribution/archives assemble --parallel
       mkdir -p ${destination}
       find distribution/archives -type f \\( -name 'elasticsearch-*-*-*-*.tar.gz' -o -name 'elasticsearch-*-*-*-*.zip' \\) -not -path *no-jdk* -exec cp {} ${destination} \\;
```

Comment:
This will need to be updated to include the docker images
```suggestion
      find distribution -type f \\( -name 'elasticsearch-*-*-*-*.tar.gz' -o -name 'elasticsearch-*-*-*-*.zip' \\) -not -path *no-jdk* -exec cp {} ${destination} \\;


```

Reply:
I think we still need to get distribution/archives here, but let me know.

Reply:
I think it will be fine if we just commit Tyler's suggestion, but I'm confirming

Reply:
Okay, this is also picking up build-context archives, do you need those? I'm guessing not.
e.g.
elasticsearch-8.0.0-SNAPSHOT-docker-build-context.tar.gz
and
elasticsearch-oss-8.0.0-SNAPSHOT-docker-build-context.tar.gz

If you don't need them, maybe add a `-not -path *build-context*` to the `find`?

Reply:
Ok done.